ui/ui-components/pages/app/[id]/edit.tsx

```diff
@@ -360,23 +377,26 @@ const Page: NextPageWithInferredProps<typeof getServerSideProps> = (props) => {
                                       Boolean(app.locked) || loadingOAuth
                                     }
                                     loading={loadingOAuth}
-                                    variant="outline-primary"
+                                    variant={"outline-primary"}
                                     onClick={() => startOAuthLogin(opt.key)}
                                   >
-                                    Sign in with OAuth
+                                    {hasValue ? "Re-request" : "Request"} Token
                                   </LoadingButton>
-
-                                  <Form.Control
-                                    className="mt-2"
-                                    required={opt.required}
-                                    type="password"
-                                    placeholder={opt.placeholder}
-                                    name={`options.${opt.key}`}
-                                    {...register(`options.${opt.key}`)}
-                                  />
-                                  <Form.Text className="text-muted">
-                                    {opt.description}
-                                  </Form.Text>
+                                  {hasValue && (
```

Comment:
This is still only being shown if it has a value. I think for filling out based on env vars you would need this shown in the empty state, too.

Reply:
ohh got it, Ok will undo that.